src/test/java/com/exadel/demo/controller/UserControllerTest.java

```diff
@@ -0,0 +1,182 @@
+package com.exadel.demo.controller;
+
+import com.exadel.demo.dto.RoleDto;
+import com.exadel.demo.dto.UserDto;
+import com.exadel.demo.service.UserService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import java.util.ArrayList;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.when;
+
+@ContextConfiguration(classes = {UserController.class})
+@ExtendWith(SpringExtension.class)
+class UserControllerTest {
+
+    @Autowired
+    private UserController userController;
+
+    @MockBean
+    private UserService userService;
+
+    @MockBean
+    private Logger logger;
+
+    @Test
+    void testGetAllUsers() throws Exception {
+        doNothing().when(this.logger).info((String) any());
+        when(this.userService.getAllUsers()).thenReturn(new ArrayList<>());
+        MockHttpServletRequestBuilder requestBuilder = MockMvcRequestBuilders.get("/user/getAll");
+        MockMvcBuilders.standaloneSetup(this.userController)
+                .build()
+                .perform(requestBuilder)
+                .andExpect(MockMvcResultMatchers.status().isOk())
+                .andExpect(MockMvcResultMatchers.content().contentType("application/json"))
+                .andExpect(MockMvcResultMatchers.content().string("[]"));
+    }
+
+    @Test
+    void testGetAllUsers2() throws Exception {
```

Comment:
better provide more descriptive name for method or use  DisplayName annotation for describe the method